articles/aws-vs-google-cloud/index.md

```diff
@@ -0,0 +1,106 @@
+
+#### Networking
+Unlike Google Cloud, AWS does not have a tiered networking framework. 
+
+#### Main Service
+The main service of Google Cloud is the Compute Engine. EC2 (Elastic Compute Cloud) is the main compute service of AWS.  
+
+#### Best feature
+AWS provides a wide variety of cloud-based services. This platform has been in operation for a longer period than Google Cloud, which has enabled it to offer reliable cloud services. Google Cloud has less experience in cloud services but provides cutting edge services in Big Data.
+
+#### Downtimes
+In 2015, both platforms experienced downtimes. Google Cloud’s downtime lasted more than 11 hours, while the maximum downtime that was experienced by AWS was two hours and 30 minutes. Both platforms were improved to reduce the chances of experiencing downtimes. The maximum downtime is an important factor when choosing a cloud service provider. 
+
+#### Pricing
+In 2017, AWS announced a [per-second billing for services such as EC2 and EBS](https://aws.amazon.com/blogs/aws/new-per-second-billing-for-ec2-instances-and-ebs-volumes/). This type of billing was extended to other services such as AWS Batch, Amazon EMR, and provisioned IOPS. In the same year, Google announced that it had extended [per-second billing](https://cloud.google.com/blog/products/gcp/extending-per-second-billing-in-google) to Cloud Dataproc, Container Engine, and Compute Engine. This clearly shows there is price competition between these services. 
+
+Google Cloud platform offers its customers discounts (Committed Use Discounts) that do not come with upfront costs. To counter this, AWS provides Reserve Instances as a way of offering discounts to its customers. Google Cloud has made an extra step in enhancing cost-saving by offering sustained user discounts.
+
+### AWS vs Google Cloud: Advantages and Disadvantages
+#### Advantages of AWS
+* AWS platform provides an organized framework of services, which enhances ease of use. 
+* Amazon Web Services provide fast and reliable cloud computing solutions.
+* AWS provides privacy for all the stored work without bugs or incidences of theft. 
+* It offers a wide variety of services, with the capability of handling many resources and users.
+  
+#### Advantages of Google Cloud
+* It provides quality computing services in Big Data, machine learning, and analytics.
+* It offers considerable load balancing and scaling.
+* It has good pricing plans. 
+* They enhance the live migration of VMs (Virtual Machines).
+
+#### Disadvantages of AWS
+* It has a long deployment process. This process can take up to 20 minutes for one website. 
+* It may not be ideal for companies that do not have a high technological capability.
+* The cost structure is difficult to understand.
+  
+#### Disadvantages of Google Cloud
+* The Google Cloud lacks certain features that exist in the AWS platform.

```

Comment:
Mentioning these features would be ideal

Reply:
Done.